Header header
string manufacturer
string board
string serial_number
string hardware_version
string firmware_version

string battery_voltage
string voltage_9v
string voltage_5v
string voltage_3v3